### DISCUSSION TITLE
Add a seed option?

### DISCUSSION THREAD

**Discussion by jaystarkey** [JPhilipp/powerdalle#3] [Ideas]
I've read about seed via the IPA - might be useful to add an option to specify the seed?

**Comment by JPhilipp** [comment]
Thanks for the suggestion! While there isn't any seed option in the API at the moment, you can copy the generated prompt (shown in second place below images, after the manual prompt), copy it into the prompt box, then tick "Exact Prompt". This will get you closer to the previous image.

If OpenAI ever adds a true seed option, I'll be sure to add it into the UI somehow!

**Comment by jaystarkey** [comment]
Oh, my mistake - I thought there was a seed option based on a post I stumbled across.  OpenAI seems determined to give minimal control to people so they don't intentionally generate images that may go against their policies.  I find for my purposes they generate the best images compared to other options, though with some frustration involved.
